### DISCUSSION TITLE
Default TaskChampion command

### DISCUSSION THREAD

**Discussion by savchenko** [taskchampion/taskchampion#157] [Ideas]
With TaskWarrior already using `task`, what do we have left?

- `tc`, abbreviates to TaskChampion, but a little clumsy to type as it requires index+thumb on a standard QWERTY keyboard.
- `ta`, which is TAsk. Less elegant as hint to TaskChampion, but much more ergonomic: small extension of left index finger towards <kbd>t</kbd> is all it takes. Left pinky usually presses <kbd>a</kbd>.

Personally, I'd go with ergonomics. Especially so considering that TC is aiming to be a full-on personal task manager. Thoughts?

**Comment by djmitche** [comment]
I type `tsak` all the time, so `ta` sounds great to me.  And in an informal survey:
```
cleave ~ $ ta
-bash: ta: command not found
```

**Reply by savchenko** [reply]
Further investigation:
```
$ apropos ^ta$
^ta$: nothing appropriate.
$ apt-cache search ta | grep "^ta\ "
$
```

This is on Debian 11.

**Comment by djmitche** [comment]
This will be fixed in #222.
